app/sidecar/base.toml

```diff
@@ -96,24 +96,24 @@ task-slots = ["sys", "packrat", { seq = "sequencer" }, "jefe"]
 [tasks.control_plane_agent]
 name = "task-control-plane-agent"
 priority = 7
-max-sizes = {flash = 131072, ram = 32768 }
-stacksize = 6144
+max-sizes = {flash = 140672, ram = 65536 }
+stacksize = 12000
```

Comment:
Do we know what doubled the stack requirement here?

Reply:
I believe this was adding the crypto verification code, which has since been pulled out into a separate branch.  We can revert the `stacksize` changes, or just leave them as-is, since we'll need it eventually.